<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>546</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>music</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images/icons.qrc">
    <normaloff>:/music.png</normaloff>:/music.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QToolButton" name="add">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
   <property name="icon">
    <iconset resource="images/icons.qrc">
     <normaloff>:/add.png</normaloff>:/add.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="play">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>390</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>播放</string>
   </property>
   <property name="icon">
    <iconset resource="images/icons.qrc">
     <normaloff>:/pause.png</normaloff>:/pause.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="stop">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
   <property name="icon">
    <iconset resource="images/icons.qrc">
     <normaloff>:/sop.png</normaloff>:/sop.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="songName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>450</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>歌曲名</string>
   </property>
  </widget>
  <widget class="QSlider" name="progress">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>480</y>
     <width>331</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="progressLabel">
   <property name="geometry">
    <rect>
     <x>381</x>
     <y>480</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>00:00/00:00</string>
   </property>
  </widget>
  <widget class="QSlider" name="volume">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>370</y>
     <width>22</width>
     <height>160</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="playbackMode">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>390</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>播放模式</string>
   </property>
   <property name="icon">
    <iconset resource="images/icons.qrc">
     <normaloff>:/sequential.png</normaloff>:/sequential.png</iconset>
   </property>
  </widget>
  <widget class="QListWidget" name="playlist">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>60</y>
     <width>191</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QToolButton" name="previous">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>390</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>上一首</string>
   </property>
   <property name="icon">
    <iconset resource="images/icons.qrc">
     <normaloff>:/previous.png</normaloff>:/previous.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="next">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>390</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>下一首</string>
   </property>
   <property name="icon">
    <iconset resource="images/icons.qrc">
     <normaloff>:/next.png</normaloff>:/next.png</iconset>
   </property>
  </widget>
  <widget class="QListWidget" name="lyrics">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>380</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="images/icons.qrc">
     <normaloff>:/th.jpg</normaloff>:/th.jpg</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="clearPlaylist">
   <property name="geometry">
    <rect>
     <x>506</x>
     <y>20</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
   <property name="icon">
    <iconset resource="images/icons.qrc">
     <normaloff>:/clear.png</normaloff>:/clear.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="images/icons.qrc">
     <normaloff>:/playlist.png</normaloff>:/playlist.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="quit">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>510</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="icon">
    <iconset resource="images/icons.qrc">
     <normaloff>:/quit.png</normaloff>:/quit.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="volumelabel">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>410</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>音量</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images/icons.qrc"/>
 </resources>
 <connections/>
</ui>
